blinky/.gitignore

```diff
@@ -1,24 +1,17 @@
-# App artifacts
-/_build
-/db
-/deps
-/*.ez
-mix.lock
-mix.lock.*
+# The directory Mix will write compiled artifacts to.
+/_build/
 
-# Nerves images
-/_images
+# If you run "mix test --cover", coverage assets end up here.
+/cover/
 
-# Generate on crash by the VM
-erl_crash.dump
+# The directory Mix downloads your dependencies sources to.
+/deps/
 
 # Where 3rd-party dependencies like ExDoc output generated docs.
-/doc
+/doc/
 
-# If you run "mix test --cover", coverage assets end up here.
-/cover
+# Ignore .fetch files in case you like to edit your project deps locally.
+/.fetch
 
-# ignore everything in rel/ except config files
-rel/*
```

Comment:
Did you intentionally un-ignore these?

Reply:
Yes. That was really old - actually, I'm not sure how old, but I've forgotten when the change happened where releases stopped being built in the rel directory.